bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
Actually this property does not have any real effect on the image apart from setting that as a property on the image, since this is simply the deploy script.

A more effective way for us would be to simply collect it from the template file's variables section where we set a default, so that we don't have to change it in these scripts whenever we implement the new Chef version. 

But I don't understand why we don't set a specific version in templates and instead just leave it as 'provisionerless'. eg. https://github.com/osuosl/packer-templates/blob/master/centos-7.3-x86_64-openstack-mitaka-aio.json#L84